brd-ios/breadwallet/src/ApplicationController.swift

```diff
@@ -144,6 +140,10 @@ class ApplicationController: Subscriber, Trackable {
     }
     
     func decideFlow() {
+        if let nvc = rootNavigationController {
```

Comment:
why now just use a coordinator who does this magic?

Reply:
how?